.github/scripts/sync-up-dotnet-ver.js

```diff
@@ -0,0 +1,149 @@
+const fse = require("fs-extra");
+const path = require("path");
+const semver = require("semver");
+const targetPath = process.env.INIT_CWD;
+const xml2js = require(path.join(targetPath, "node_modules/xml2js"));
+// ---- target pkg name and version -----
+const pkgName = process.env.npm_package_name;
+const pkgVer = process.env.npm_package_name;
```

Comment:
should it be process.env.npm.**package_ver**?

Reply:
no, node set it as npm_package_name
ref:
https://www.ruanyifeng.com/blog/2016/10/npm_scripts.html